week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Tutaj powinien być dodany jeszcze:

```
dist
.parcel-cache
```
To katalogi których nie powinno być na repo :)

`dist` jest generowany z automatu, natomiast `.parcel-cache` też i jest potrzebny tylko w `dev`